src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubBuildStatusNotification.java

```diff
@@ -67,12 +67,20 @@
  *
  */
 public class GitHubBuildStatusNotification {
-    
+
     private static final Logger LOGGER = Logger.getLogger(GitHubBuildStatusNotification.class.getName());
 
-    private static void createCommitStatus(@Nonnull GHRepository repo, @Nonnull String revision, @Nonnull GHCommitState state, @Nonnull String url, @Nonnull String message, @Nonnull Job<?,?> job) throws IOException {
+    private static void createCommitStatus(@Nonnull GHRepository repo, @Nonnull SCMRevision revision, @Nonnull GHCommitState state, @Nonnull String url, @Nonnull String message) throws IOException {
         LOGGER.log(Level.FINE, "{0}/commit/{1} {2} from {3}", new Object[] {repo.getHtmlUrl(), revision, state, url});
-        repo.createCommitStatus(revision, state, url, message, "Jenkins job " + job.getName());
+        String status_context;
+        String revisionToNotify = resolveHeadCommit(repo, revision);
+
+        if (revision instanceof PullRequestSCMRevision) {
+            status_context = "continuous-integration/jenkins/pr";
+        } else {
+            status_context = "continuous-integration/jenkins/branch";
```

Comment:
Wait, is it actually a problem if both `buildForkPRHead` and `buildForkPRMerge` have identical status_contexts?

I guess, there is potentially a window (assuming you have 1 executor) when it shows green from the first to run, then goes yellow as the second builds...  But that behavior, while not ideal, is better than it is now.

Currently, you can't use github status checks at all due to the constantly changing names.